### PR TITLE
Fix isort config to not skip __init__.py files (but few)

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [x] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

Some files were forgotten during last isort pass, because isort ignores `__init__.py` files per default.

# Solution

Fix config file to include those but few exceptions.

# Action

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -19,5 +19,6 @@ multi_line_output=3
 order_by_type=True
 sections=FUTURE,STDLIB,THIRDPARTY,QT,LOCALFOLDER,RESOURCES,PICARD_UI
 skip_glob=**/picard/ui/ui_*.py
-skip=./tagger.py,picard/const/attributes.py,picard/const/countries.py,picard/resources.py,scripts/picard.in
+skip=./tagger.py,picard/const/attributes.py,picard/const/countries.py,picard/resources.py,scripts/picard.in,picard/const/__init__.py,picard/formats/__init__.py
```
Comment (phw) [inline .isort.cfg]: What's the issue with `formats/__init__.py` and `const/__init__.py`? Shouldn't it be possible to sort the imports to the start, too?

**Comment by phw** [comment]
Sorry for breaking the tests, we need https://github.com/metabrainz/picard/pull/956 to fix that

File: .isort.cfg (hunk 1 above)
Comment (zas) [reply .isort.cfg]: I just excluded problematic ones for now, either due to style or just crappy coding like caa providers

**Comment by zas** [comment]
> Sorry for breaking the tests, we need #956 to fix that

Yes, i'll rebase once  #956 is in master

**Pull request merged**